docs/core-concepts/run-udfs/run.mdx

```diff
@@ -251,6 +251,20 @@ A [`fused.run()`](/python-sdk/top-level-functions/#run) call will require the fo
 3. [Optional] Any protected arguments as seen in the dedicated [API docs page](/python-sdk/top-level-functions/#run) (if applicable). These include for example:
    - `bounds` -> A geographical bounding box (as a list of 4 point: `[min_x, min_y, max_x, max_y]`) defining the area of interest.
    - `cache_max_age` -> The maximum [age of the UDF's cache](/core-concepts/cache/#defining-your-cache-lifetime-cache_max_age).
+   - `dtype_out_raster` -> The output raster format. Defaults to "tif". Change this to "png" for a simple preview of your image.
+   - `dtype_out_vector` -> The output vector format. Defaults to "parquet".
```

Comment:
note that these parameters are only relevant when running remotely, local runs I don't think go through the full serialization/deserialization process